internal/cephfs/nodeserver.go

```diff
@@ -270,6 +275,9 @@ func (ns *NodeServer) NodeStageVolume(
 		}
 	}
 
+	err = ns.healthChecker.StartChecker(req.GetVolumeId(), stagingTargetPath)
+	log.WarningLog(ctx, "failed to start healthchecker: %v", err)
```

Comment:
this should be done inside `if err !=nil` check

Reply:
done!